app/cypress/integration/cif/project-revision/edit-project-revision.spec.js

```diff
@@ -118,7 +118,7 @@ describe("when editing a project, the project page", () => {
     cy.findByText(/No reports due/).should("be.visible");
     cy.contains("Changes saved.");
 
-    cy.wait(1000);
+    cy.findByText(/Complete/i).should("be.visible");
```

Comment:
Does the test still work if we add a `should('be.visible')` assertion to line 119 instead? (or replace with `cy.findByText(/Changes saved/i).should("be.visible");`) 

If we ever change the test so the status is something other than complete, we'll have to update this line too.

Reply:
Further to our conversation, this line gives enough time for Axe to get run correctly and not raise accessibility issues on the due date status badge.